src/meshapi/views/map.py

```diff
@@ -213,6 +214,79 @@ def list(self, request: Request, *args: List[Any], **kwargs: Dict[str, Any]) ->
 
         response.data.extend(cable_runs)
 
+        # Since the old school map has no concept of a LOS, only potential Links, we need to
+        # create a fake potential Link object to represent each of our LOS entries
+        # For our purposes here, we only care about LOS entries between buildings that have
+        # install numbers. If one side of an LOS is a building that has no installs associated with
+        # it, we exclude it
+        los_objects_with_installs = (
+            LOS.objects.filter(
+                Exists(Install.objects.filter(building=OuterRef("from_building")))
+                & Exists(Install.objects.filter(building=OuterRef("to_building")))
+                & ~Q(from_building=F("to_building"))
+            )
+            .exclude(
+                # Remove any LOS objects that would duplicate Link objects,
+                # to avoid cluttering the map
+                Exists(
+                    Link.objects.filter(
+                        (
+                            Q(from_device__node__buildings=OuterRef("from_building"))
+                            & Q(to_device__node__buildings=OuterRef("to_building"))
+                        )
+                        | (
+                            Q(from_device__node__buildings=OuterRef("to_building"))
+                            & Q(to_device__node__buildings=OuterRef("from_building"))
+                        )
+                    )
+                )
+            )
+            .filter(
```

Comment:
"It works™, we have Tests ;)"
- Andrew, 2024

Reply:
I stand by this